ci/jobs/scripts/check_style/aspell-ignore/en/aspell-dict.txt

```diff
@@ -2637,8 +2637,8 @@ SchemaInferenceMode
 schemas
 Schemas
 Schwartzian
-searchAll
-searchAny
+hasAnyTokens
+hasAllTokens
```

Comment:
aspell dict entries are expected to be alphabetically sorted.

Reply:
For 99% of ClickHouse's existence, the entries were sorted in ASCII order ABC...abc, e.g. [here in 28.8](https://github.com/ClickHouse/ClickHouse/blob/v25.8.2.29-lts/ci/jobs/scripts/check_style/aspell-ignore/en/aspell-dict.txt).

Someone destroyed the order recently, and it is now AaBbCc...

I'll make a follow-up PR which restores the correct order. Until then, all effort to add new entries in a sorted order are wasted.